web/src/main/java/com/epam/rft/atsy/web/controllers/CandidateCreationController.java

```diff
@@ -2,38 +2,54 @@
 
 import com.epam.rft.atsy.service.CandidateService;
 import com.epam.rft.atsy.service.domain.CandidateDTO;
+import com.epam.rft.atsy.web.FileUploadingProperties;
+import com.epam.rft.atsy.web.model.file.FileStatus;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.servlet.ModelAndView;
 
 import javax.annotation.Resource;
+import javax.servlet.http.HttpSession;
 
 @Controller
-@RequestMapping(path = "/secure")
+@RequestMapping(path = "/secure/candidate")
 public class CandidateCreationController {
 
-  public static final String CANDIDATE_OBJECT_KEY = "candidate";
+  private static final String CANDIDATE_OBJECT_KEY = "candidate";
   private static final String VIEW_NAME = "candidate_create";
+
   @Resource
   private CandidateService candidateService;
 
-  @RequestMapping(method = RequestMethod.GET, path = "/candidate/{candidateId}")
-  public ModelAndView loadCandidate(@PathVariable(value = "candidateId") Long candidateId) {
-    ModelAndView modelAndView = new ModelAndView(VIEW_NAME);
 
+  @RequestMapping(method = RequestMethod.GET, path = "/{candidateId}")
+  public ModelAndView loadCandidate(@PathVariable(value = "candidateId") Long candidateId,
+                                    HttpSession session) {
+    ModelAndView modelAndView = new ModelAndView(VIEW_NAME);
     modelAndView.addObject(CANDIDATE_OBJECT_KEY, candidateService.getCandidate(candidateId));
 
+    String cvPath = session.getAttribute(FileUploadingProperties.SESSION_PARAM_CV_PATH) == null ?
```

Comment:
There could be a method for this in FileUploadingProperties (it could be renamed to FileUploadingUtil)